docs/source/release_notes/lts/52_0.rst

```diff
@@ -0,0 +1,162 @@
+  skip decorators instead.
+- Sysinfo collection based on command execution now allows a timeout to
+  be set.
+- The non-local runner plugins, the html plugin and the yaml_to_mux
+  plugin are now distributed in separate packages.
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
+- Support for new type of test status, ``CANCEL``, and of course the
+  mechanisms to set a test with this status.
+- ``avocado.TestFail``, ``avocado.TestError`` and ``avocado.TestCancel``
+  are now public Avocado Test APIs, available from the main avocado
+  namespace.
+- Introduction of the robot plugin, which allows Robot Framework tests
+  to be listed and executed natively within Avocado.
+- A brand new ResultsDB optional plugin.
+- Listing of supported loaders (``--loaders \?``) was refined.
+- Variant-IDs generated by yaml_to_mux plugin now include leaf node
+  names to make them more meaningful, making easier to skim through
+  the results.
+- `yaml_to_mux` now supports internal filters defined inside the YAML
+  file expanding the filtering capabilities even further.
+- Avocado now supports resuming jobs that were interrupted.
+- The HTML report now presents the test ID and variant ID in separate
+  columns, allowing users to also sort and filter results based on those
+  specific fields.
+- The HTML report will now show the test parameters used in a test when
+  the user hovers the cursor over the test name.
+- Avocado now reports the total job execution time on the UI, instead of
+  just the tests execution time.
+- The Avocado multiplex command has been renamed to variants.
+- Loaders were tweaked to provide more info on ``avocado list -V``
+  especially when they don't recognize the reference.
+- Users can use ``--ignore-missing-references on`` to run a job with
+  undiscovered test references
+- Users can now choose in which order the job will execute tests (from
+  its suite) and variants. The two available options are
+  ``--execution-order=variants-per-test`` (default) or
+  ``--execution-order=tests-per-variant``.
+- Test methods can be recursively discovered from parent classes by upon
+  the ``:avocado: recursive`` docstring directive.
+
+Those are the main, but not all, features that are available to users.
+Besides that, we had several improvements in our ``utils`` libraries
+that are important for test writers, some of them are listed below:
+
+- ``time_to_seconds``, ``geometric_mean`` and ``compare_matrices`` were
+  added in ``avocado.utils.data_structures``.
+- ``avocado.utils.distro`` was refined.
+- ``avocado.utils.{filelock,lv_utils,multipath,partition,pci}`` were\
+  introduced.
+- ``avocado.utils.memory`` contains several new methods.
+- New ``avocado.utils.process.SubProcess.get_pid()`` method.
+- ``sudo`` support in ``avocado.utils.process`` was improved
+- The ``avocado.utils.process`` library makes it possible to ignore
+  spawned background processes that never finish.
```

Comment:
well they can finish, `avocado.utils.process` just does not care and finish immediately (I'd suggest removing the `that never finish`)